docs/ref/contrib/admin/admindocs.txt

```diff
@@ -12,14 +12,16 @@ docstrings of models, views, template tags, and template filters for any app in
 :setting:`INSTALLED_APPS` and makes that documentation available from the
 :mod:`Django admin <django.contrib.admin>`.
 
-You may, to some extent, utilize :mod:`~django.contrib.admindocs` to quickly
```

Comment:
Could you revert this documentation change? I think it merely restores an older version of the text (at least I don't see any substantive changes).